besskge/sharding.py

```diff
@@ -48,14 +48,14 @@ class Sharding:
     @property
     def n_entity(self) -> int:
         """
-        Number of entities in the KG.
+        Number of entities in the knowledge graph.
         """
         return len(self.entity_to_shard)
 
     @property
     def max_entity_per_shard(self) -> int:
         """
-        Number of entities (including possibily one final padding) in each shard.
+        Number of entities (including possibly one final padding) in each shard.
```

Comment:
I realized there was a small mistake in the implementation of the padding, [I pushed a fix for this](https://github.com/graphcore-research/bess-kge/pull/17) - as a result this docstring was modified, so you might need to rebase (sorry!)